docs/how-to-use-minio-server-side-encryption-with-aws-cli.md

```diff
@@ -1,88 +1,94 @@
-
- ## 1. Prerequisites
-
-Install Minio Server **with TLS** from [here](https://docs.minio.io/docs/how-to-secure-access-to-minio-server-with-tls).
-
-Notice that tools like aws-cli or mc will show an error if use a self-signed TLS certificate and try to upload objects to the server. Please take a look at Let's Encrypt to get a CA-signed TLS certificate. Self-signed certificates should only be used for development/testing or internal usage.
-
-## 2. Use SSE-C with the aws-cli
-
-Install the aws-cli like shown [here](https://docs.minio.io/docs/aws-cli-with-minio).
-
-Let's assume your running a local minio server on `https://localhost:9000` with
-a self-signed certificate. To skip the TLS certificate verification you need to
-specify: `--no-verify-ssl`. If your minio server uses a CA-signed certificate you
-should **never** specify `--no-verify-ssl`. Otherwise the aws-cli would accept
-any certificate. 
-
-### 2.1 Upload an object.
-
-1. Create a bucket named `my-bucket`:  
-`aws --no-verify-ssl --endpoint-url https://localhost:9000 s3api create-bucket --bucket my-bucket`
-2. Upload an object using SSE-C. The object name is `my-secret-diary` and the
-   its content is the file `~/my-diary.txt`.
-    ```
-    aws s3api put-object \
-    --no-verify-ssl \
-    --endpoint-url https://localhost:9000 \
-    --bucket my-bucket --key my-secret-diary \
-    --sse-customer-algorithm AES256 \
-    --sse-customer-key MzJieXRlc2xvbmdzZWNyZXRrZXltdXN0cHJvdmlkZWQ= \
-    --sse-customer-key-md5 7PpPLAK26ONlVUGOWlusfg== \
-    --body ~/my-diary.txt
-    ```
-    You should use your own encryption key.
-
-### 2.2 Show object information
-  You **must** specify the correct SSE-C key of an encrypted object to show its metadata:
-  ```
-  aws s3api head-object \
+# How to use Minio's Server-side Encryption with the AWS CLI
+
+Minio supports S3 server-side-encryption with customer provided keys (SSE-C). This following sections describe this use of this encryption with the AWS Command Line Interface (`awscli`):
+* [Prerequisites](#prerequisites)
+* [Use SSE-C with aws-cli](#use-sse-c-with-aws-cli)
+* [Security Notice](#security-notice)
+
+# <a name="prerequisites"></a>Prerequisites
+
+A client must specify three HTTP headers for SSE-C requests:
+* `X-Amz-Server-Side-Encryption-Customer-Algorithm`: The algorithm identifier: must be set to `AES256`.
+* `X-Amz-Server-Side-Encryption-Customer-Key`: The secret encryption key: must be a 256-bit Base64-encoded string.
+* `X-Amz-Server-Side-Encryption-Customer-Key-MD5`: The encryption key MD5 checksum: must be set to the MD5-sum of the encryption key. Note: The MD5 checksum is the MD5 sum of the raw binary key, not of the base64-encoded key.
+
+Install the Minio Server with TLS as described [here](https://docs.minio.io/docs/how-to-secure-access-to-minio-server-with-tls).
```

Comment:
I like the numbered 1/2/3 steps format.

Reply:
@deekoder I have fixed line 3. Please let me know if other changes are desired.